### ISSUE TITLE
Content becomes empty when pasting

### ISSUE THREAD

**Issue by steftrenson** [fraywing/textAngular#512]
Hi,

we've found an issue when pasting content from another application into textangular. 
```html
<div class="AppContainer" style="width: 1280px;color: rgb(0, 0, 0);">
	<div id="c_base" class="c_base">
		<div id="c_content" class="c_main">
			<div id="pageContent">
				<div id="pageInbox" class="v-Page">
					<div id="inboxControl0f">
						<div class="containsYSizerBar" style="height: 849px;width: 1280px;">
							<div class="ContentRight WithRightRail FullView">
								<div class="ContentRightInner t_mbgc t_qtc t_urtc" style="color: rgb(68, 68, 68);background-color: rgb(255, 255, 255);">
									<div id="inboxControl0fv-ReadMessageContainer" class="v-ReadMessageContainer slideOnResize">
										<div class="c-ReadMessage" style="height: 818.03125px;width: 895px;">
											<div class="rmMessages ClearBoth" id="ReadMessageScrollableSection">
												<div id="readMessagePartControl1604f" class="c-ReadMessagePart ReadMsgContainer HasLayout ClearBoth HideShadows FullPart NoHistory Read RmIc">
													<div class="c-ReadMessagePartBody">
														<div class="readMsgBody">
															<div id="bodyreadMessagePartBodyControl1609f" class="ExternalClass MsgBodyContainer">
																<p>
																	<u>
																		<b>
																			Lorem ipsum
																		</b>
																	</u>
																</p>
																<p>
																	<b>Lorem ipsum</b>
																</p>
																<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
															</div>
														</div>
													</div>
												</div>
											</div>
										</div>
									</div>
								</div>
							</div>
						</div>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>
```
When pasting the above example into textangular, the text appears to be fine. But when the focus is removed from textangular, the text disappears. We suspect the problem is the many nested div's.

Do you have any idea why this problem occurs?

I'm using v1.3.0-23 on Google Chrome 40.0.2214.91

**Comment by SimeonC** [comment]
Wow, that's some nested code!
Yes I do know why this occurs - it's the [_blankTest function](https://github.com/fraywing/textAngular/blob/master/lib/taBind.js#L2-L19) which checks to see if there is any content or not. I had some limiters on it to help when there is a lot of text but it seems that I missed some edge cases like this.

I'll get this patched up and let you know when it's fixed.

**Comment by SimeonC** [comment]
Ok, should be patched and fixed in v1.3.3 just released.
